Resources/Prototypes/explosion.yml

```diff
@@ -42,6 +42,17 @@
   texturePath: /Textures/Effects/fire_greyscale.rsi 
   fireStates: 3
 
+- type: explosion
+  id: PureRadioactive
```

Comment:
this doesnt exactly follow the naming conventions, It would normally be RadioactivePure